framework/packages/abstract-client/src/account.rs

```diff
@@ -182,18 +190,23 @@ impl<'a, Chain: CwEnv> AccountBuilder<'a, Chain> {
         verifiers::validate_description(self.description.as_deref())?;
         verifiers::validate_link(self.link.as_deref())?;
 
-        let abstract_account = self.abstr.account_factory.create_new_account(
-            AccountDetails {
-                name,
-                description: self.description.clone(),
-                link: self.link.clone(),
-                namespace: self.namespace.as_ref().map(ToString::to_string),
-                base_asset: self.base_asset.clone(),
-                install_modules: vec![],
-            },
-            ownership,
-            Some(&[]),
-        )?;
+        let account_details = AccountDetails {
+            name,
+            description: self.description.clone(),
+            link: self.link.clone(),
+            namespace: self.namespace.as_ref().map(ToString::to_string),
+            base_asset: self.base_asset.clone(),
+            install_modules: vec![],
+        };
+        let abstract_account = if let Some(owner_account) = self.owner_account {
+            owner_account
+                .abstr_account
+                .create_sub_account(account_details, Some(&[]))?
```

Comment:
We should probably add a `.funds` field in the builder as well. But lets do that in a different PR.